README.md

```diff
@@ -56,7 +56,7 @@ Add the following commands to your `.gitlab-ci.yml`:
 ```yaml
 before_script:
   - sudo pip install https://github.com/kosma/art
-  - art configure <url> <token>
+  - art configure <url> --token-type <private|job> <token>
```

Comment:
This syntax isn't consistent. `<foo>` is a placeholder for an argument - see `<token>` right next to it. I think you want `{private,job}` or maybe `{"private","job"}`.

Reply:
Agreed. Went with the former.